addons/spreadsheet/static/src/pivot/plugins/pivot_odoo_core_plugin.js

```diff
@@ -20,6 +20,24 @@ export class PivotOdooCorePlugin extends OdooCorePlugin {
         }
     }
 
+    import(data) {
+        if (data.pivots) {
+            for (const id in data.pivots) {
+                // Ensure that the sorted column is a measure
+                // and if not, drop it.
+                // This situation can happen because of a bug in a previous version
```

Comment:
Shouldn't it be a migration then ? What if there are revisions whith this bug ?Or we could check the sanity of the sortedColumn at the pivot creation, it would probably make more sense than here no ?

Reply:
totally forgot revisions :man_facepalming: